tests/unit/executors/indexers/test_binary_indexer.py

```diff
@@ -30,3 +30,111 @@ def validate(req):
         d.ClearField('embedding')
     with f:
         f.search(docs_no_embedding, output_fn=validate, override_doc_id=False)
+
+
+def test_binarypb_update(tmpdir):
```

Comment:
use test_metas fixture in our conftest.py

Reply:
This test is too long. Could you divide it in different ones? I see comments that suggest that different behaviors are being tested and I think is better if they are reflected in a separate test

Reply:
sure